css/index.css

```diff
@@ -46,25 +49,36 @@ h1 {
   line-height: 1.4;
   
   */
-  
+  .content-row p{
+    font-size: 16px;
+    font-family: roboto;
+    padding: 10px;
+    line-height: 1.4;  
+  }
```

Comment:
The comments call out for a selector that styles ALL p tags. 
This is a selector that styles all p tags that nested inside of .content-row.